README.md

```diff
@@ -435,6 +435,21 @@ go mod tidy
 go mod vendor
 ```
 
+## Building without network acess
+
+Go modules require network acess. If you need to make a repetible build with
+no network acess, make sure that your code is under `$GOPATH` and run the
+following:
+
+``shell
+mkdir -p /tmp/gopath/src/github.com/u-root/
+cd /tmp/gopath/src/github.com/u-root/
+git clone https://github.com/u-root/u-root.git
+cd u-root
+GO111MODULE=off GOPATH=/tmp/gopath GOPROXY=off go build
```

Comment:
I just went to my linode node, and did this, and it worked as you describe on lines 449-450, 
so the sum total of my commands (and, note, I did NOT need to mkdir /tmp/go):
```shell
export GOPATH=/tmp/go
GO111MODULE=off go get github.com/u-root/u-root
cd $GOPATH/src/github.com/u-root/u-root
 GO111MODULE=off GOPROXY=off go build
 GO111MODULE=off GOPROXY=off ./u-root
```

If you want, you might try these and make sure I did not mess something up.

So when you write this up, you might say:
Pick a location for your off-network build, for example, but it can be anywhere. The directory does not need to exist ahead of time. 
For this example, we'll use /tmp/go

Also, /tmp/go is maybe a better name because people might think it has to be /tmp/gopath to match GOPATH -- you'd be amazed at how much people read into these examples :-)

Reply:
actually, to really make the point, using a name like
export GOPATH=/tmp/$$
would really make the point even more, possibly.
Something to make it clear that go doesn't need to be in the name.